test/parallel/test-net-connect-options-port.js

```diff
@@ -27,35 +27,37 @@ const net = require('net');
 
 // Test wrong type of ports
 {
-  function portTypeError(opt) {
-    const prefix = '"port" option should be a number or string: ';
-    const cleaned = opt.replace(/[\\^$.*+?()[\]{}|=!<>:-]/g, '\\$&');
-    return new RegExp(`^TypeError: ${prefix}${cleaned}$`);
+  function portTypeError() {
```

Comment:
IMHO simply add a commit, since it makes re-reviews easier.
A common practice is to prefix the new commit's message with `[fixup] ...` so it's clear that it should be squashed when the PR is landed.